features/shared/auto_remove_outdated_config.feature

```diff
@@ -6,6 +6,9 @@ Feature: automatically remove outdated git-town configuration
 
 
   Scenario: automatically remove outdated branch hierarchy information
-    Given I run `git-town hack feature; git checkout main; git branch -d feature`
-    When I run `git-town sync`
-    Then Git Town has no branch hierarchy information
+    Given I run "git-town hack feature"
+    And I run "git checkout main"
+    And I run "git branch -d feature"
+    When I run "git-town sync"
+    And Git Town is now aware of this branch hierarchy
+      | BRANCH | PARENT |
```

Comment:
This feels confusing. Thoughts on `Git Town has no branch hierarchy information`?

Reply:
Agree. Implemented